.github/workflows/auditbeat-macos-unit-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: auditbeat-macos-unit-tests
+
+on:
+  pull_request:
```

Comment:
Is the path also missed in the `pull-request` event? IIUC, this will run always regardless.